src/Symfony/Component/Security/Http/Tests/Firewall/RememberMeListenerTest.php

```diff
@@ -248,10 +313,11 @@ protected function getListener($withDispatcher = false, $catchExceptions = true)
             $manager = $this->getManager(),
             $logger = $this->getLogger(),
             $dispatcher = ($withDispatcher ? $this->getDispatcher() : null),
-            $catchExceptions
+            $catchExceptions,
+            $sessionStrategy = $this->getSessionStrategy()
```

Comment:
@xabbuh, hm, it would be nice to have a test for this behaviour, I think. I modified it to leave other tests untouched, please have a look.